apps/api/src/city/dto/create-city.dto.ts

```diff
@@ -0,0 +1,32 @@
+import { ApiProperty } from '@nestjs/swagger/dist/decorators'
+import { Expose } from 'class-transformer'
+import { IsNumber, IsOptional, IsPositive, IsString, IsUUID, Max } from 'class-validator'
+import { Prisma } from '.prisma/client'
+
+export class CreateCityDto {
+  @ApiProperty()
+  @Expose()
+  @IsString()
+  name: string
+
+  @ApiProperty()
+  @IsOptional()
+  @Expose()
+  @IsNumber()
+  @IsPositive()
+  @Max(500000)
```

Comment:
Please use the validator `@IsPostalCode(locale?: string)` 

Not sure `500000` is the actual max for the postcode, let's just use the postal code validator for now

Check the docs at https://github.com/typestack/class-validator#validation-decorators